entrypoint.sh

```diff
@@ -6,7 +6,8 @@ PORT="8000"
 set -e
 
 if [ "$1" = 'server' ]; then
-    gunicorn server:api -b :$PORT --chdir armada/api
```

Comment:
Could the documentation also be updated to include the new required volume mounts?